index.html

```diff
@@ -41,11 +41,9 @@ <h2>Discuss</h2>
 
 
 <section class="container">
-    <h1>Latest News{% comment %} <a href="https://groups.google.com/forum/#!forum/ritlug-announce"><i class="fas fa-envelope icon-orange"></i></a> <a href="/feeds/latest.xml"><i class="fas fa-rss-square icon-orange"></i></a>{% endcomment %}</h1>
```

Comment:
The first link here should go to the mailing list, it's a mail icon...

Reply:
This change is intentional. For a consistent user experience, I wanted to keep the user on the site because the mailing list would not have context here. When they click here, they will find other announcements already posted on the site.

Reply:
@jwflory if you look at the deployed site, it's a mail icon. My point is that it's misleading/confusing to users and it something else would be more appropriate

Reply:
@ct-martin Any suggestions?

Reply:
@jwflory hide the icon and add a button below the card. Should probably file an issue for this